tests/destination_details_mock_test.go

```diff
@@ -69,8 +69,10 @@ func prepareDestinationDetailsResponse() string {
 				"password": "%s"
 			}
 		}
-	}`, DESTINATION_DETAILS_ID, DESTINATION_DETAILS_ID, DESTINATION_DETAILS_SERVICE, DESTINATION_DETAILS_REGION, DESTINATION_DETAILS_TIME_ZONE, DESTINATION_DETAILS_SETUP_STATUS, DESTINATION_DETAILS_HOST, DESTINATION_DETAILS_PORT, DESTINATION_DETAILS_DATABASE, DESTINATION_DETAILS_AUTH, DESTINATION_DETAILS_USER, DESTINATION_DETAILS_MASKED_PASSWORD)
-	return responce
+	}`, DESTINATION_DETAILS_ID, DESTINATION_DETAILS_ID, DESTINATION_DETAILS_SERVICE,
```

Comment:
Please every item in the new row, like this:
DESTINATION_DETAILS_ID,
DESTINATION_DETAILS_ID,
DESTINATION_DETAILS_SERVICE,
DESTINATION_DETAILS_REGION,
DESTINATION_DETAILS_TIME_ZONE,